modules/gravatar-hovercards.php

```diff
@@ -12,7 +12,7 @@
  * Additional Search Queries: gravatar, hovercards
  */
 
-define( 'GROFILES__CACHE_BUSTER', gmdate( 'YM' ) . '-2' ); // Break CDN cache, increment when gravatar.com/js/gprofiles.js changes.
+define( 'GROFILES__CACHE_BUSTER', gmdate( 'YM' ) . '-3' ); // Break CDN cache, increment when gravatar.com/js/gprofiles.js changes.
```

Comment:
What would you think about switching to a weekly cache, like we do for other similar resources in Jetpack, like Stats? This way we wouldn't have to worry about incrementing anything here, things would get updated week after week.

```suggestion
define( 'GROFILES__CACHE_BUSTER', gmdate( 'YW' ) );
```

Reply:
That sounds good to me! This file isn't so large that it needs monthly caching, nor does it get updated too often, so a weekly cache sounds like a good middle-ground 👍